source/contracts/personal_edition.md

```diff
@@ -0,0 +1,93 @@
+# Safe Personal Edition Smart Contract
+
+This version is targeted at users that control all keys owning a safe. The transaction hash can be signed with the private keys that manage the safe. 
```

Comment:
**S**afe

Reply:
Also, I think this is confusing "This version is targeted at users that control all keys owning a safe. "  I think it has to be rephrase in a way to say that a single user controls all keys.

Reply:
Something like "This version is targeted at individual users, i.e. only single user would have access to all keys owning a Safe."